spring-cloud-azure-autoconfigure/pom.xml

```diff
@@ -1,177 +1,158 @@
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>spring-integration-servicebus</artifactId>
-            <optional>true</optional>
-        </dependency>
-
-        <!-- Redis -->
-        <dependency>
-            <groupId>org.springframework.data</groupId>
-            <artifactId>spring-data-redis</artifactId>
-            <optional>true</optional>
-        </dependency>
-
-        <!-- Kafka -->
-        <dependency>
-            <groupId>org.springframework.kafka</groupId>
-            <artifactId>spring-kafka</artifactId>
-            <optional>true</optional>
-        </dependency>
-
-        <!-- MongoDB -->
-        <dependency>
-            <groupId>org.springframework.data</groupId>
-            <artifactId>spring-data-mongodb</artifactId>
-            <optional>true</optional>
-        </dependency>
-
-        <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-context-support</artifactId>
-            <optional>true</optional>
-        </dependency>
-
-        <!--Spring JMS-->
-        <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-jms</artifactId>
-            <optional>true</optional>
-        </dependency>
-
-        <!--Qpid-->
-        <dependency>
-            <groupId>org.apache.qpid</groupId>
-            <artifactId>qpid-jms-client</artifactId>
-            <optional>true</optional>
-        </dependency>
-
-    </dependencies>
-
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jar-plugin</artifactId>
-                <version>3.0.2</version>
-                <configuration>
-                    <archive>
-                        <manifest>
-                            <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
-                            <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
-                        </manifest>
-                    </archive>
-                </configuration>
-            </plugin>
-        </plugins>
-        <resources>
-            <!-- Raw Resource -->
-            <resource>
-                <directory>src/main/resources</directory>
-                <includes>
-                    <include>META-INF/spring.factories</include>
-                </includes>
-            </resource>
-            <resource>
-                <directory>src/main/resources</directory>
-                <filtering>true</filtering>
-                <includes>
-                    <include>telemetry.config</include>
-                </includes>
-            </resource>
-        </resources>
-    </build>
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<groupId>com.microsoft.azure</groupId>
+		<artifactId>spring-cloud-azure</artifactId>
```

Comment:
I suggest that we revert changes in this file and create another commit since the indent of 4 spaces makes the pom.xml difficult to review.

Reply:
You can use "Ignore whitespace changes" to eliminate that: 
![image](https://user-images.githubusercontent.com/7320197/85967297-270fd680-b990-11ea-9f83-d106b9f726af.png)

Reply:
![image](https://user-images.githubusercontent.com/7320197/86179713-cb149180-baf8-11ea-9072-7a37b5c14b93.png)